packages/adp-tooling/src/writer/options.ts

```diff
@@ -178,21 +180,28 @@ function addOpenSourceMiddlewares(ui5Config: UI5Config, config: AdpWriterConfig)
  * @returns list of required tasks.
  */
 function getAdpCloudCustomTasks(config: AdpWriterConfig & { target: AbapTarget } & { app: CloudApp }): CustomTask[] {
+    const configuration: CloudCustomTaskConfig = {
+        type: 'abap',
+        destination: config.target?.destination,
+        appName: config?.app?.bspName,
+        languages: config?.app?.languages?.map((language: Language) => {
+            return {
+                sap: language.sap,
+                i18n: language.i18n
+            };
+        })
+    };
+
+    if (!isAppStudio()) {
```

Comment:
Could you always add this please, so that projects are ready for usage everywhere.
Even if you run in AppStudio, you have the url and auth type. The url is part of the destination configuration and the authtype is always `reentranceTicket` since we only add the task for ABAP cloud systems.

Reply:
For BAS we don't need this configuration, because we are using destinations.

Reply:
A project generated in BAS and then checked into e.g. github, could be checked out in VSCode and it should also work there as well. Just because it is generated in BAS, does not necessarily mean it only used in BAS.

Reply:
Fixed in [a3c9caa](https://github.com/SAP/open-ux-tools/pull/2362/commits/a3c9caada35cc2c1ce209f4e86e7b03a04f73ba5)

Reply:
Maybe I am misreading the code (and the snapshots) but as far as I can see the url is still not being added when a project is generated in BAS. Please ensure that if you generate in BAS, the project also works in VSCode.

Reply:
Fixed in [5467cec](https://github.com/SAP/open-ux-tools/pull/2362/commits/5467cec391857a90519c5c396b8eab7edc06088d)